bottom/unpack.py

```diff
@@ -300,6 +300,21 @@ def unpack_command(msg: str) -> Tuple[str, Dict[str, Any]]:
         else:
             kwargs["message"] = ""
 
+    elif command in ["MODE"]:
+        nickmask(prefix, kwargs)
+        if "#" in params[0]:
+            command = "CHANNELMODE"
+            kwargs["channel"] = params[0]
+            kwargs["modes"] = params[1]
+            if len(params) > 2:
+                kwargs["params"] = params[2:]
+            else:
+                kwargs["params"] = ""
```

Comment:
Shouldn't we have an empty list here so `kwargs["params"]` is always a list?  I think (sincerely hope) that's the current convention.

Reply:
That's fine, I'll change this to an empty list to keep the convention.